docs/troubleshooting/node_export.md

```diff
@@ -0,0 +1,29 @@
+# Reporting a Node Bug
+
+When it comes to node issues, it is best to export specific information, which can help us debug and help with the problem
+
+## Exporting node details
```

Comment:
I would use just one title `Export node json`  and then create `###` for sub steps

Reply:
Done! :)